docs/reference/changelog-r2021.md

```diff
@@ -22,6 +22,7 @@ Released on June, Xth, 2021.
     - Allowed the `wb_supervisor_node_reset_physics` function to reset the physics of solid descendants of the given node ([#2742](https://github.com/cyberbotics/webots/pull/2742)).
   - New Samples:
     - Added a simple room with a Nao robot ([#2701](https://github.com/cyberbotics/webots/pull/2701)).
+    - Added a generic gear proto generator and a demo showing it being used in a collision-based transmission ([#2805](https://github.com/cyberbotics/webots/pull/2805)).
```

Comment:
We usually don't use the word "generator" although all the procedural protos are somewhat generators...
```suggestion
    - Added a generic gear proto and a demo showing it being used in a collision-based transmission ([#2805](https://github.com/cyberbotics/webots/pull/2805)).
```